index.js

```diff
@@ -1,4 +1,10 @@
 import "./styles.css";
 import $ from "jquery";
 
-$(document.body).append("<h1>Hello World!</h1>");
+let content = `
+<div class="content">
+	<img id="logo" src="//stealjs.com/img/steal-landing-page-logo.jpg">
```

Comment:
Let’s use this SVG so it looks nice on high-DPI screens: http://stealjs.com/docs/static/node_modules/bit-docs-docjs-theme/static/img/StealJS-Logo-V2.svg

I’m going to make the same change on the home page: https://github.com/stealjs/stealjs/issues/30